katana/native/cases/static/cases/js/casesApp.js

```diff
@@ -0,0 +1,1040 @@
+
+		bid = "addTestStepAbove-"+s+"-id-";
+		items.push('<i title="Insert" class="fa fa-plus" theSid="'+s+'"   id="'+bid+'" katana-click="caseApp.addCaseFromLine()" key="'+bid+'"/>');
+
+		bid = "dupTestStepAbove-"+s+"-id-";
+		items.push('<i title="Duplicate" class="fa fa-copy" theSid="'+s+'"  id="'+bid+'" katana-click="caseApp.duplicateCaseFromLine()" key="'+bid+'"/>');
+
+	}
+
+	items.push('</tbody>');
+	items.push('</table>'); // 
+	katana.$activeTab.find("#tableOfTestStepsForCase").html( items.join(""));
+	katana.$activeTab.find('#Step_table_display tbody').sortable( { stop: caseApp.testCaseSortEventHandler});
+	
+	caseApp.fillCaseDefaultGoto();
+	katana.$activeTab.find('#default_onError').on('change',caseApp.fillCaseDefaultGoto );
+
+	/*
+	if (caseApp.jsonCaseDetails['Datatype'] == 'Custom') {
+		$(".arguments-div").hide();
+	} else {
+
+		$(".arguments-div").show();
+	}
+	*/
+	
+	}, // end of function 
+
+	deleteCaseFromLine : function() {
+	var names = this.attr('key').split('-');
+	var sid = parseInt(names[1]);
+	caseApp.removeTestStep(sid);
+	},
+
+	editCaseFromLine: function() { 
+	var names = this.attr('key').split('-');
+	var sid = parseInt(names[1]);
+	katana.popupController.open(katana.$activeTab.find("#editCaseStepDiv").html(),"Edit..." + sid, function(popup) {
+		katana.$activeTab.find("#editCaseStepDiv").attr('popup-id',popup);
+		caseApp.setupPopupDialog(sid,caseApp.jsonCaseSteps['step'] ,popup);
+	});
+	},	
+
+	addCaseFromLine: function() {
+	var names = this.attr('key').split('-');
+	var sid = parseInt(names[1]);
+	caseApp.addTestStepAboveToUI(sid,caseApp.jsonCaseSteps['step'],0);
+	},
+
+	duplicateCaseFromLine :function() { 
+		var names = this.attr('key').split('-');
+		var sid = parseInt(names[1]);
+		caseApp.addTestStepAboveToUI(sid,caseApp.jsonCaseSteps['step'],1);
+	},
+
+	makePopupArguments: function(popup,  oneCaseStep) {
+		caseApp.lastPopup = popup; 
+		var a_items = [] ;
+		var xstr;
+		var bid;
+		var arguments = oneCaseStep['Arguments']['argument'];
+		var ta = 0; 
+		var sid = parseInt(popup.find("#StepRowToEdit").attr('value'));	
+		
+		for (xarg in arguments) {
+				//console.log(arguments[xarg]);
+			a_items.push('<div class="row">');
+			a_items.push('<label class="col-md-2">Name</label><input  type="text" argid="caseArgName-'+ta+'" value="'+arguments[xarg]["@name"]+'"/>');
+			a_items.push('<label class="col-md-2">Value</label><input  type="text" argid="caseArgValue-'+ta+'" value="'+arguments[xarg]["@value"]+'"/>');
+			// Now a button to edit or delete ... 
+			bid = "deleteCaseArg-"+sid+"-"+ta+"-id"
+			a_items.push('<td><i title="Delete" class="fa fa-eraser" value="X" id="'+bid+'" key="'+bid+'"  katana-click="caseApp.deletePopupArgument"/>');
+			
+			bid = "saveCaseArg-"+sid+"-"+ta+"-id"
+			a_items.push('<td><i  title="Save Argument Change" class="fa fa-pencil" value="Save" id="'+bid+'" key="'+bid+'"  katana-click="caseApp.savePopupArgument"/>');
```

Comment:
Change fa-pencil to this http://fontawesome.io/icon/floppy-o/

Reply:
Done